lib/Xamarin.Forms.Vonage.Android/Xamarin.Forms.Vonage.Android.csproj

```diff
@@ -65,19 +67,17 @@
     <None Include="Resources\AboutResources.txt" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Android.Support.v7.AppCompat" Version="28.0.0.3" />
+    <PackageReference Include="Xamarin.AndroidX.AppCompat" Version="1.0.2-preview01" />
     <PackageReference Include="Xamarin.Forms">
-      <Version>5.0.0.2515</Version>
+      <Version>5.0.0.2578</Version>
```

Comment:
if we are updating XF version here, let's update it in the entire solution (all projects)